index.js

```diff
@@ -0,0 +1,108 @@
+// import Book from modules
+import addToLocalStorage from './modules/localStorage.js';
+import Book from './modules/shelfClass.js';
+
+let books;
+
+const book1 = new Book('title1', 'author1', '001');
+const book2 = new Book('title2', 'author2', '002');
+const book3 = new Book('title3', 'author3', '003');
+
+books = [book1, book2, book3];
+
+// add event listner to newly added book remove button
+let removeBtn = document.querySelectorAll('.remove');
+
+function removeBook(ref) {
+  const result = books.filter((value) => value.id !== ref);
+  books = result;
```

Comment:
- [ ] Please, consider using arrow function syntax, as required in the project (use ES6 syntax). For example, here you can go on with 
```
const removeBook = (ref) => {
....
```

- [ ] Also, will be better to create an additional module for removing the books method, then import it here in `index.js` file